code/game/machinery/mech_recharger.dm

```diff
@@ -110,4 +110,22 @@
 
 /obj/machinery/mech_recharger/proc/stop_charging()
 	update_use_power(1)
-	charging = null
\ No newline at end of file
+	charging = null
+
+
+/obj/machinery/mech_recharger/hephaestus
```

Comment:
might be worth looking into making these rechargers not charge a exosuit if it's over 95% charge to prevent it from constantly draining power (in another PR)